app/payloads/api/v1/patient_payload.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::PatientPayload
```

Comment:
Does this need a constructor? (looking at how it's called up above)

Reply:
The constructor is provided by ActiveModel::Model